### DISCUSSION TITLE
Event for wire:poll

### DISCUSSION THREAD

**Discussion by Restingo** [livewire/livewire#3244] [1. Help] [answered]
Is there an javascript event that gets fired when wire:poll polled something?

**Comment by joshhanley** [comment, marked as answer]
@Restingo you could listen for `message.sent` event hook for when the polling message is sent, or `message.received` for when it comes back. Check out the Javascript Hooks section here in the docs https://laravel-livewire.com/docs/2.x/lifecycle-hooks#:~:text=cycle-,message.sent,-Called

Hope this helps!

**Reply by Restingo** [reply]
Thank you, that works!
